verify GREEN token

DEEPAKTRIPATHHI05 opened the pull request:
yaml
name: "GREEN"
symbol: "GREEN"
decimals: 9
address: "EQAVDt4jtvTeelQmCqkhuPLyjMRO_EwUEqFV0zoBG1ZQkmiH"
icon: "https://greenmeme.art/websitecssassets/images/banner-img.png"
description: "A MISSION TO EDUCATE BILLIONS OF PEOPLE ABOUT THE IMPORTANCE OF TREES AND THE IMPACT OF DEFORESTRATION."
social_links:
  - website: "https://greenmeme.art/"
  - telegram: "https://t.me/greenmemtoken"

DEEPAKTRIPATHHI05 committed:
Update STORM.yaml

DEEPAKTRIPATHHI05 committed:
"main"

DEEPAKTRIPATHHI05 committed:
Create GREEN.yaml

green.yaml added

DEEPAKTRIPATHHI05 commented:
Hi Tonkeeper Team,

I submitted this Pull Request on [28/12/24]. Could you please provide an update on its status? Let me know if any further information or changes are needed.

Thank you for your time and assistance!

Best regards,
DEEPAKTRIPATHHI

DEEPAKTRIPATHHI05 committed:
Delete jettons/STORM.yaml

DEEPAKTRIPATHHI05 commented:
Hi Tonkeeper Team,

I hope you’re doing well. I am following up again regarding Pull Request #1905 that I submitted on [28/12/24] and followed up on [29/12/24]. Could you please provide an update on the current status or let me know if further actions are needed from my side?

Thank you for your assistance and understanding!

Best regards,
DEEPAKTRIPATHHI

Irso007 commented:
1. The image link does not open. Fix this.

2. There are some issues with this request:

ERROR:root:failed to get megaton assets
Traceback (most recent call last):
  File "/home/runner/work/ton-assets/ton-assets/generator.py", line 1[7](https://github.com/tonkeeper/ton-assets/pull/1905/checks#step:5:8)6, in <module>
    main()
  File "/home/runner/work/ton-assets/ton-assets/generator.py", line 11[8](https://github.com/tonkeeper/ton-assets/pull/1905/checks#step:5:9), in main
    jettons = merge_jettons()
              ^^^^^^^^^^^^^^^
  File "/home/runner/work/ton-assets/ton-assets/generator.py", line 63, in merge_jettons
    raise Exception("invalid keys %s in %s" % (set(j.keys()) - ALLOWED_KEYS, j.get('name')))
Exception: invalid keys {'social_links', 'icon'} in GREEN

DEEPAKTRIPATHHI05 committed:
Update GREEN.yaml

DEEPAKTRIPATHHI05 commented:
Hi team,
I have resolved the mentioned issues:

Fixed the image link issue.
Addressed the error with invalid keys (social_links and icon) in the GREEN token configuration.
Could you please review the updates and let me know if further changes are needed?

Thank you!

Irso007 commented:
Incorrect data in pull request
